ipaserver/install/dogtag.py

```diff
@@ -23,3 +29,31 @@ class DogtagInstallInterface(service.ServiceInstallInterface):
     )
     ca_file = prepare_only(ca_file)
     ca_file = replica_install_only(ca_file)
+
+    pki_config_override = knob(
+        str, None,
+        cli_names='--pki-config-override',
+        description="Path to ini file with config overrides.",
+    )
+
+    @pki_config_override.validator
+    def pki_config_override(self, value):
+        if value is None:
+            return
+        if not os.path.isabs(value):
+            raise ValueError("must use an absolute path")
+        try:
+            cfg = RawConfigParser()
+            with open(value) as f:
+                cfg.readfp(f)  # pylint: disable=deprecated-method
+        except Exception as e:
+            raise ValueError(
+                "Invalid config '{}': {}".format(value, e)
+            )
+
+    token_password = knob(
```

Comment:
We most likely want a `--token-password-file` option, instead of or in addition to `--token-password`, so that it is possible to convey the password with being visible in `/proc` and similar?